aspnetcore/blazor/progressive-web-app.md

```diff
@@ -52,6 +52,74 @@ dotnet new blazorwasm -o MyNewProject --pwa
 
 Optionally, PWA can be configured for an app created from the ASP.NET Core Hosted template. The PWA scenario is independent of the hosting model.
 
+## Convert an existing Blazor WebAssembly app into a PWA
+
+Convert an existing Blazor WebAssembly app into a PWA following the guidance in this section.
+
+In the app's project file:
+
+* Add the following `ServiceWorkerAssetsManifest` property to a `PropertyGroup`:
+
+  ```xml
+    ...
+    <ServiceWorkerAssetsManifest>service-worker-assets.js</ServiceWorkerAssetsManifest>
+  </PropertyGroup>
+   ```
+
+* Add the following `ServiceWorker` item to an `ItemGroup`:
+
+  ```xml
+  <ItemGroup>
+    <ServiceWorker Include="wwwroot\service-worker.js" 
+      PublishedContent="wwwroot\service-worker.published.js" />
+  </ItemGroup>
+  ```
+
+::: moniker range=">= aspnetcore-5.0"
+
+Navigate to the ASP.NET Core GitHub repository at the following URL, which links to 5.0 release reference source and assets. If you aren't converting an app for the 5.0 release, select the release that you're working with from the **Switch branches or tags** drop-down list that applies to your app.
```

Comment:
Maybe instead of referring people to the source code we can ask them to use the `dotnet new` interface and copy the assets from there?

Seems better to me than having people dig through GH.

Reply:
I don't think we've ever taken that approach, but we could offer it.

> dig through GH

They don't need to traverse any folders. They just need to select the branch. A certain amount of developer training is probably ok. It looks like GH is something now that all .NET devs working above hobbyist level will need to know a little bit about it. This technique has been used successfully for a while. It doesn't generate doc issues indicating confused developers.

I'll modify the passage to provide both approaches.